tests/test_processing.py

```diff
@@ -101,6 +106,18 @@ def test_remove_inner_chunks(
             20,
             [UnknownChunk(0x5, 0x8), UnknownChunk(0xA, 0xF)],
         ),
+        pytest.param(
```

Comment:
not sure we want to solve the case when there are overlapping chunks. This is a problem in the chunk handler and by handling overlapping chunks, we would hid potential handler bugs.